Component/SolidMechanics/FEM/Elastic/src/sofa/component/solidmechanics/fem/elastic/TetrahedronFEMForceField.inl

```diff
@@ -1349,6 +1349,12 @@ void TetrahedronFEMForceField<DataTypes>::init()
         if (youngModulus[i]>maxYoung) maxYoung=youngModulus[i];
     }
 
+    const Real& poissonRatio = _poissonRatio.getValue();
+    if (poissonRatio<0 || poissonRatio>=0.5)
+    {
+        msg_warning() << "FEM Poisson Ratio in Hooke's law should be in [0,0.5[.";
+    }
```

Comment:
```suggestion
    if (poissonRatio < 0 || poissonRatio >= 0.5)
    {
        msg_warning() << "FEM Poisson's Ratio in Hooke's law should be in [0,0.5[.";
    }
```

Reply:
The two choice should be either:
- a warning + clamping the value to the closest boundary
- an error and setting the component to invalid

But please don't mix warning with invalid component state.